Week-1/assignment-1.2/style.css

```diff
@@ -0,0 +1,119 @@
+* {
+   font-family: 'League Spartan', sans-serif;
+   overflow: hidden;
+}
+
+#asideicons {
+   list-style-type: none;
+   margin: 3px;
+   padding: 0;
+   width: 3%;
+   background-color: #f1f1f1;
+   position: fixed;
+   height: 100%;
+}
+
+li {
+   display: block;
+   color: #000;
+   padding: 35px 16px;
+   text-decoration: none;
+}
+
+nav {
+   margin-left: 15px;
+   height: 315px;
+   display: flex;
+   background-color: #383838;
+   color: white;
+   width: 100%;
+
+}
+
+#img {
+   padding: 60px;
+   padding-left: 100px;
+   height: 200px;
+   width: 200px;
+}
+
+#navsec {
+   margin: 45px;
+}
+#navsec h1{
+   font-size: 25px;
+}
+
+#navsec h6 {
+   margin: 0px;
+   margin-top: 0px;
+   padding: 2px;
+}
+
+.playicon {
+   display: inline;
+   margin: 20px;
```

Comment:
try to use consistent padding and margin in al the places and it should be in multiple of 8 and even number not odd